service.js

```diff
@@ -58,7 +58,7 @@ require('./migrate-psql-db.js')(function (err) {
               logger: log.logger
             });
   seneca.use(require('./oauth2.js'), {clients: config.oauth2.clients});
-  seneca.use('user');
+  seneca.use('user', { failedCount: 3 });
```

Comment:
options seems to be "failedLoginCount" rather than failedCount 
https://github.com/senecajs/seneca-user/blob/8e26f8515fbbc51955096fe707e389c13eaee165/default-options.json#L10